ui/v2.5/src/components/Scenes/SceneCard.tsx

```diff
@@ -0,0 +1,248 @@
+      const markerPopover = { ...marker, scene: { id: props.scene.id } };
+      return <TagLink key={marker.id} marker={markerPopover} />;
+    });
+
+    return (
+      <HoverPopover placement="bottom" content={popoverContent}>
+        <Button className="minimal">
+          <Icon icon="map-marker-alt" />
+          <span>{props.scene.scene_markers.length}</span>
+        </Button>
+      </HoverPopover>
+    );
+  }
+
+  function maybeRenderOCounter() {
+    if (props.scene.o_counter) {
+      return (
+        <div>
+          <Button className="minimal">
+            <SweatDrops />
+            <span>{props.scene.o_counter}</span>
+          </Button>
+        </div>
+      );
+    }
+  }
+
+  function maybeRenderPopoverButtonGroup() {
+    if (
+      props.scene.tags.length > 0 ||
+      props.scene.performers.length > 0 ||
+      props.scene.scene_markers.length > 0 ||
+      props.scene?.o_counter
+    ) {
+      return (
+        <>
+          <hr />
+          <ButtonGroup className="scene-popovers">
+            {maybeRenderTagPopoverButton()}
+            {maybeRenderPerformerPopoverButton()}
+            {maybeRenderSceneMarkerPopoverButton()}
+            {maybeRenderOCounter()}
+          </ButtonGroup>
+        </>
+      );
+    }
+  }
+
+  function onMouseEnter() {
+    if (!previewPath || previewPath === "") {
+      setPreviewPath(props.scene.paths.preview || "");
+    }
+    VideoHoverHook.onMouseEnter(videoHoverHook);
+  }
+  function onMouseLeave() {
+    VideoHoverHook.onMouseLeave(videoHoverHook);
+    setPreviewPath("");
+  }
+
+  function isPortrait() {
+    const { file } = props.scene;
+    const width = file.width ? file.width : 0;
+    const height = file.height ? file.height : 0;
+    return height > width;
+  }
+
+  let shiftKey = false;
+
+  return (
+    <Card
+      className={`scene-card zoom-${props.zoomIndex}`}
+      onMouseEnter={onMouseEnter}
+      onMouseLeave={onMouseLeave}
+    >
+      <Form.Control
+        type="checkbox"
+        className="scene-card-check d-none d-sm-block"
+        checked={props.selected}
+        onChange={() => props.onSelectedChanged(!props.selected, shiftKey)}
+        onClick={(event: React.MouseEvent<HTMLInputElement, MouseEvent>) => {
+          // eslint-disable-next-line prefer-destructuring
+          shiftKey = event.shiftKey;
+          event.stopPropagation();
+        }}
+      />
+      {maybeRenderSceneStudioOverlay()}
+      <Link to={`/scenes/${props.scene.id}`} className="scene-card-link">
+        {maybeRenderRatingBanner()}
+        {maybeRenderSceneSpecsOverlay()}
+        <video
+          loop
+          className={cx("scene-card-video", { portrait: isPortrait() })}
+          poster={props.scene.paths.screenshot || ""}
+          ref={videoHoverHook.videoEl}
+        >
+          {previewPath ? <source src={previewPath} /> : ""}
+        </video>
+      </Link>
+      <div className="card-section">
+        <h5 className="text-truncate">
```

Comment:
The old version didn't truncate the title text. Is there a reason why we're truncating it now?

Reply:
v2 truncates in the same way: https://github.com/stashapp/stash/blob/develop/ui/v2/src/components/scenes/SceneCard.tsx#L240

Reply:
The v2 is missing `white-space: nowrap`, which prevents the truncation (and rendering the other two styles superfluous). So v2.5 successfully truncates, while v2 does not. In my opinion, not truncating is better.

Reply:
Ah, I think I see what's happening. Filenames without whitespace are being truncated in both versions, but titles with whitespace are only being truncated in 2.5. I was just testing with scene titles. I'll remove the whitespace no-wrap.